apps/passport-server/test/zuconnect.spec.ts

```diff
@@ -157,39 +157,36 @@ describe("zuconnect functionality", function () {
     try {
       expect(await zuconnectTripshaSyncService.sync()).to.throw;
     } catch (e) {
-      /**
-       * We get detailed validation errors here due to the schema defined in
-       * {@link ZuconnectTripshaSchema}
-       */
-      expect((e as any).issues[0].code).to.eq("invalid_type");
-      expect((e as any).issues[0].path).to.deep.eq(["tickets"]);
-      expect((e as any).issues[0].expected).to.eq("array");
-      expect((e as any).issues[0].received).to.eq("undefined");
+      expect(e instanceof Error).to.be.true;
     }
 
     // Test that validation fails for a ticket with no email address
     server.use(makeHandler(badTicketsResponse));
     try {
-      expect(await zuconnectTripshaSyncService.sync()).to.throw;
+      expect(await zuconnectTripshaSyncService.sync()).to.not.throw;
+      const tickets = await fetchAllZuconnectTickets(db);
+      expect(tickets.length).to.eq(0);
     } catch (e) {
-      expect((e as any).issues[0].code).to.eq("invalid_type");
-      expect((e as any).issues[0].path).to.deep.eq(["tickets", 0, "email"]);
-      expect((e as any).issues[0].expected).to.eq("string");
-      expect((e as any).issues[0].received).to.eq("undefined");
+      // Should never get here
+      expect(false).to.be.true;
```

Comment:
i think you can do something like `expect.fail`, but u should check before making that change, if you make the change